### ISSUE TITLE
Can't use multiple labels file in single `rules_oci` rule

### ISSUE THREAD

**Issue by bozaro** [bazel-contrib/rules_oci#608]
I want to add two kind of labels:
 - labels for image lifecycle management - on this labels change image must be rebuild;
 - labels for image stamping (from which Git revision I can build same image) - this lables change everytime and should be excluded from cache key.

For this I want to create two labels file and pass it by custom or `filegroup` rule:
 - `labels.txt` - with labels for image lifecycle management;
 - `labels_volatile.txt` - with stamping labels.

After that I can add `labels_volatile.txt` to `omitted_inputs` srubbing rule and cache will ignore every `labels_volatile.txt` file change.

But currently I can pass only single file by `labels` attribute to `rules_oci`.

**Comment by thesayyn** [comment]
I don't quite understand the feature request here. How's label_list instead of label will help you with this? 

Also this is possible today by using two oci_image targets, first one adding labels.txt and the second one adding volatile.txt. 

IMO this is really a custom feature that the most will not benefit from.

**Comment by bozaro** [comment]
I understand that most people are not interested in the problem of stamping on the farm.

But in this case, support is added by a trivial non-breaking change: https://github.com/bazel-contrib/rules_oci/pull/609

**Comment by thesayyn** [comment]
I agree that looks simple in terms of added lines, however it makes the public api a little more harder to understand.  This is possible to today by combining two oci_image with little-to-no-cost at build time.
